src/debugging/netcore/NetCoreDebugHelper.ts

```diff
@@ -287,20 +292,32 @@ export class NetCoreDebugHelper implements DebugHelper {
         return pathNormalize(result, platformOS);
     }
 
-    private async copyDebuggerToContainer(containerName: string, containerDebuggerDirectory: string): Promise<string> {
+    private async copyDebuggerToContainer(context: DockerDebugContext, containerName: string, containerDebuggerDirectory: string, containerOS: ContainerOSType): Promise<void> {
+        const dockerClient = new CliDockerClient(new ChildProcessProvider());
+        if (containerOS === 'windows') {
+            const isolation = await dockerClient.inspectObject(containerName, { format: '{{ .HostConfig.Isolation}}' });
+            if (isolation && isolation === 'hyperv') {
+                context.actionContext.errorHandling.suppressReportIssue = true;
+                throw new Error(localize('vscode-docker.debug.netcore.isolationNotSupported', 'Unable to copy debugger. This operation is not supported on Hyper-V container.'));

```

Comment:
If this message is supposed to be seen by the user, maybe it would be better to refer to what they were trying to do. Something like

    Attaching a debugger to a Hyper-V container is not supported.